### PR TITLE
TECH-10 - Grafana Cloud + PEP8 linting + tests fix/adjustments

### PR DESCRIPTION
**Pull request by OleksandrUA**
1) Adding the ability to send metrics to Grafana Cloud with Grafana key and endpoint passed as an argument
2) Linting according to PEP8 standard
3) Fixing all tests as tests expected asserts are no longer valid in 2022 and tests didn't cover `--alias`, `-graphite-key` and  `-graphite-endpoint` arguments

### CODE REVIEW

**Review by eskp — APPROVED** [review]

File: petrometer/petrometer.py [hunk 1]
```diff
@@ -41,19 +41,51 @@
...
+            metavar="ADDRESSES",
+            nargs="+",
+            type=str,
+            help="Ethereum addresses to get the gas usage of",
+        )
+        parser.add_argument(
+            "--etherscan-api-key", help="Etherscan API key", required=True, type=str
+        )
+        parser.add_argument(
+            "--ethfiller-key", help="Graphite API key", required=True, type=str
```
Comment (cristidas) [inline petrometer/petrometer.py]: @OleksandrUA why is this flag called `--ethfiller-key` if you're feeding it `GRAPHITE_KEY`? Probably something like `--graphite-key` should be more accurate.
Also, I think this tool should be used also if there is no need to send data to Graphite, so probably `required=False` would make more sense.

Comment (OleksandrUA) [reply petrometer/petrometer.py]: @cristidas 
1. It was called this (`--ethfiller-key`) way back in 2020 (see `/home/ubuntu/petrometer/ethfiller.sh` on the instance). So I decided not to change any naming which are there for 2 years. 
2. The value which was replaced by `--ethfiller-key` was initially just hard-coded in the script and as script relies on this value I made `--ethfiller-key` mandatory. Script's logic so far is to send _all_ new transactions to Graphite and in script's logic I don't see ways to use it without `--ethfiller-key` argument.

So considering that ticket was not about re-factoring the script but rather removing drift between VCS and instance I think these changes are good to go.

File: petrometer/petrometer.py [hunk 2]
```diff
@@ -41,19 +41,51 @@
...
-        parser = argparse.ArgumentParser(prog='petrometer')
-        parser.add_argument("addresses", metavar='ADDRESSES', nargs='+', type=str,
-                            help="Ethereum addresses to get the gas usage of")
-        parser.add_argument("--etherscan-api-key", help="Etherscan API key", required=True, type=str)
-        parser.add_argument("-j", '--json', help="Generate result as JSON", dest='json', action='store_true')
-        parser.add_argument("-o", "--output", help="File to save the output to", required=False, type=str)
-        parser.add_argument("-i", "--incoming", help="Show incoming transaction gas usage, defaut outgoing",
-                            required=False, action='store_true')
+        parser = argparse.ArgumentParser(prog="petrometer")
+        parser.add_argument(
```
Comment (cristidas) [inline petrometer/petrometer.py:48]: @OleksandrUA I see formatting for some of the parsing vars has changed. Can you make them all look the same, so we have consistency?

File: petrometer/petrometer.py [hunk 3]
```diff
@@ -41,19 +41,51 @@
...
+            type=str,
+            help="Ethereum addresses to get the gas usage of",
+        )
+        parser.add_argument(
+            "--etherscan-api-key", help="Etherscan API key", required=True, type=str
+        )
+        parser.add_argument(
+            "--ethfiller-key", help="Graphite API key", required=True, type=str
+        )
+        parser.add_argument("--alias", help="Address alias", required=True, type=str)
```
Comment (cristidas) [inline petrometer/petrometer.py]: @OleksandrUA should `--alias` be mandatory if it's used only for sending data to Graphite?

File: README.md [hunk 4]
```diff
@@ -30,7 +31,7 @@ pip3 install -r requirements.txt
 ## Usage
 
 ```
-usage: petrometer [-h] --etherscan-api-key ETHERSCAN_API_KEY [-j] [-o OUTPUT]
+usage: petrometer [-h] --etherscan-api-key ETHERSCAN_API_KEY --ethfiller-key GRAPHITE_KEY [-j] [-o OUTPUT]
```
Comment (cristidas) [inline README.md]: @OleksandrUA can you add also `--alias` please?

File: petrometer/petrometer.py [hunk 5]
```diff
@@ -219,48 +334,83 @@ def get_db(address: str):
...
+            return True
+
+        # As --graphite-key is not mandatory
+        # checking if it was set
+        if not graphite_key:
+            return True
+
+        result = requests.request(
+            method="POST",
+            url="https://graphite-us-central1.grafana.net/metrics",
```
Comment (cristidas) [inline petrometer/petrometer.py]: @OleksandrUA Should this be passed as an argument instead? I think the endpoint could be different for different people.

File: README.md [hunk 6]
```diff
@@ -30,7 +31,7 @@ pip3 install -r requirements.txt
 ## Usage
 
 ```
-usage: petrometer [-h] --etherscan-api-key ETHERSCAN_API_KEY [-j] [-o OUTPUT]
+usage: petrometer [-h] --etherscan-api-key ETHERSCAN_API_KEY --graphite-key GRAPHITE_KEY --graphite-endpoint GRAPHITE_ENDPOINT --alias SOME_ALIAS [-j] [-o OUTPUT]
```
Comment (cristidas) [inline README.md]: @OleksandrUA can you denote here the 3 new flags as being optional?

Comment (OleksandrUA) [reply README.md]: @cristidas right, missed this one, thanks. Please check - fixed

**Review by cristidas — APPROVED** [review]

**Pull request merged**